csv2Urls.py

```diff
@@ -1,13 +1,16 @@
 import csv
 
+#path to csv
 csv_Path = 'eis-listing.csv'
+
+#Base url
 base_HTML_EPA_url = 'https://cdxnodengn.epa.gov/cdx-enepa-II/public/action/eis/details?eisId='
 
 with open(csv_Path) as csvfile:
     reader = csv.DictReader(csvfile)
-    text_file = open("EISIds.txt", "w")
+    text_file = open("EIS_URLs.txt", "w")
```

Comment:
Maybe we uncouple the code from EPA/EIS? And we call this "paths.txt" to be inline with our other tools?